fpsim/locations/data_processing/breastfeeding_stats.R

```diff
@@ -0,0 +1,20 @@
+### Data to process DHS data for breastfeeding duration
+
+dir <- "directory" ## directory with data
+filename <- "filename" ## must be a KR (child recode) file from the DHS for automatic processing
+
+##Breastfeeding duration 
+setwd(dir)
+dat <- read.dta13(filename)
+
+require(ggplot2)
+
+dat_nonmiss <- subset(dat, m5 < 93) #m5 = months of breastfeeding in previous pregnancy
```

Comment:
This is the line I'm wondering if was the same for Senegal.  It seems like we want a distribution of who is _still _breastfeeding__ by month, not total months of breastfeeding

Reply:
Yep, this is the same for Senegal. This parameter gives a fixed duration of breastfeeding, not a probability each month of continuing to breastfeed. So, we wouldn't want a cumulative distribution of continuing to breastfeed unless we were checking each month postpartum and assessing a probability of continuing to breastfeed.

Reply:
Yes, sorry, you're right -- we're assigning a length of breastfeeding.  The difference in shapes of the density histograms between Senegal and Kenya was throwing me off.   I'm surprised that most women in Senegal breastfeed < 6 months total duration when the mean is 20 months.